src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = (random.nextInt(Color.values().length));
+        return Color.values()[index].name();
+    }
+
+    private enum Color {
+        red,
+        orange,
+        yellow,
+        green,
+        blue,
+        purple
```

Comment:
Enum names should be written in uppercase letters. This is a common convention to make them stand out as constants.